src/zm_ffmpeg.h

```diff
@@ -47,6 +47,12 @@ extern "C" {
 #else
 #include <libavcodec/opt.h>
 #endif
+    
+#if LIBAVUTIL_VERSION_CHECK(54, 6, 0, 6, 0)
+#include <libavutil/imgutils.h>
+#else
```

Comment:
Making me stretch to remember why it ended up like this.
1. ffmpeg/avutil.h <> libavutil/avutil.h. But actually it is already included above so it can go.
2. Didn't need imgutils.h until this version when avpicture_fill was deprecated in avcodec, so just the if will be fine.
imgutils.h includes avutil.h.
Lucky I need to fix spacing anyway.